### PR TITLE
user guides: Review user documentation for Apps.

### PR DESCRIPTION
**Pull request by Ivche1337**

### CODE REVIEW

**Comment by Ivche1337** [comment]
https://codein.withgoogle.com/dashboard/task-instances/4869953071087616/

**Comment by eeshangarg** [comment]
@Ivche1337: Your comments are concise and useful! Thanks for working on this! Merged! :)

File: user-guides/ivche1337/Apps.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   * Easy to follow
+   * Well described steps
+   * Installed it on Linux
+   * No typos
+## Using Zulip on Windows
+   * The documentation explains how the Zulip Windows app works,
+   and it describes how it's updated regularly, and everything
+   linked to the app is layed out well.
+   * The documentation explains how to connect to a server,
+   i connected to a server following the instructions,
```
Comment (lonerz) [inline user-guides/ivche1337/Apps.md]: r/i/I

File: user-guides/ivche1337/Apps.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   and i didn't have any problem.
```
Comment (lonerz) [inline user-guides/ivche1337/Apps.md]: r/i/I

File: user-guides/ivche1337/Apps.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+   * Screenshots for "Connecting to a server" would be usefull
+## Using Zulip on Android
+   * Intalled it on androd
```
Comment (lonerz) [inline user-guides/ivche1337/Apps.md]: r/Intalled/Installed

r/androd/Android

File: user-guides/ivche1337/Apps.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+   * Connetcting to a server well explained (screenshots would be usefull)
```
Comment (lonerz) [inline user-guides/ivche1337/Apps.md]: r/Connetcting/Connecting

Comment (lonerz) [reply user-guides/ivche1337/Apps.md]: r/usefull/useful

**Pull request merged**